modules/enumeratum/src/main/scala/pureconfig/module/enumeratum.scala

```diff
@@ -0,0 +1,30 @@
+package pureconfig.module
+
+import _root_.enumeratum.values._
+import _root_.enumeratum._
+import pureconfig.ConfigConvert
+
+import scala.util.Try
+
+package object enumeratum {
+  implicit def enumeratumConfigConvert[A <: EnumEntry](implicit enum: Enum[A]): ConfigConvert[A] =
+    ConfigConvert.stringConvert[A](name => Try(enum.withName(name)), _.toString)
+
+  implicit def enumeratumIntConfigConvert[A <: IntEnumEntry](implicit enum: IntEnum[A]): ConfigConvert[A] =
+    ConfigConvert.stringConvert[A](v => Try(enum.withValue(v.toInt)), _.toString)
```

Comment:
The second argument to `stringConvert` is for converting the `A` back into a value which can be written to a config file. I don't think `toString` is going to be the correct conversion for that. I think you need something like `_.value.toString`.
Ditto for the various converter instances below.
(I could be totally off base. I'm just looking at Enumeratum for the first time now.)